### ISSUE TITLE
Context menu entry changes after updating for the first time from the store

### ISSUE THREAD

**Issue by rubenalamina** [microsoft/terminal#11531]
### Windows Terminal version (or Windows build number)

1.10.2714

### Other Software

The context menu entry changes from Open in Windows Terminal to Open in Windows Terminal Preview after it updates from the store for the first time. I'm running Windows 11 stable build 22000.258 after doing a clean installation. 

[About screenshot](https://i.imgur.com/JfE75sG.png)
![image](https://user-images.githubusercontent.com/18356694/137809341-d4f4719a-9d84-4a47-bd7c-3467f62f2b37.png)

[Context menu entry](https://i.imgur.com/41KDqXv.png) after it updates for the first time.
![image](https://user-images.githubusercontent.com/18356694/137809330-6a215197-de92-4a64-b78c-5b4b073187eb.png)

### Steps to reproduce

- Perform a clean installation of Windows 11 and upon reaching the desktop for the first time, right lick on it to see the context menu entry.
- Open the MS Store and check for updates. Terminal updates for the first time.
- The context menu entry will change to **Open in Windows Terminal Preview** even when the version it updated from is the stable release.

### Expected Behavior

No changes to the context menu entry.

### Actual Behavior

Context menu entry changes to **Open in Windows Terminal Preview** even though it's the stable release.

**Comment by DHowett** [comment]
I'm guessing the loc folks may have recycled the keys here. Let me check a source pull from toubhdown...

**Comment by DHowett** [comment]
```
Project     Name                                          Value
-------     ----                                          -----
TerminalApp ShellExtension_OpenInTerminalMenuItem         Öffnen in Windows-Terminal
TerminalApp ShellExtension_OpenInTerminalMenuItem         Abrir en Terminal Windows en vista previa
TerminalApp ShellExtension_OpenInTerminalMenuItem         Ouvrir dans Windows Terminal
TerminalApp ShellExtension_OpenInTerminalMenuItem         Apri in Terminale Windows
TerminalApp ShellExtension_OpenInTerminalMenuItem         Windows ターミナルで開く
TerminalApp ShellExtension_OpenInTerminalMenuItem         Windows 터미널에서 열기
TerminalApp ShellExtension_OpenInTerminalMenuItem         Abrir no Terminal do Windows
TerminalApp ShellExtension_OpenInTerminalMenuItem         Ŏφêη ιņ Windows Τĕѓмĭⁿдŀ !!! !!! !
TerminalApp ShellExtension_OpenInTerminalMenuItem         Ōρεñ ìņ Windows Ťėѓмìⁿãĺ !!! !!! !
TerminalApp ShellExtension_OpenInTerminalMenuItem         Öрĕη ϊŉ Windows Ţėѓмìņαľ !!! !!! !
TerminalApp ShellExtension_OpenInTerminalMenuItem         Открыть в Терминале Windows
TerminalApp ShellExtension_OpenInTerminalMenuItem         在 Windows 终端 中打开
TerminalApp ShellExtension_OpenInTerminalMenuItem         使用 Windows 終端機開啟
TerminalApp ShellExtension_OpenInTerminalMenuItem_Preview Öffnen in Windows-Terminal Vorschau
TerminalApp ShellExtension_OpenInTerminalMenuItem_Preview Abrir en Terminal Windows en vista previa
TerminalApp ShellExtension_OpenInTerminalMenuItem_Preview Ouvrir dans l’aperçu Terminal Windows
TerminalApp ShellExtension_OpenInTerminalMenuItem_Preview Apri nell'anteprima di Terminale Windows
TerminalApp ShellExtension_OpenInTerminalMenuItem_Preview Windows ターミナル (プレビュー) で開く
TerminalApp ShellExtension_OpenInTerminalMenuItem_Preview Windows 터미널 미리 보기에서 열기
TerminalApp ShellExtension_OpenInTerminalMenuItem_Preview Abrir na Visualização do Terminal do Windows
TerminalApp ShellExtension_OpenInTerminalMenuItem_Preview Ōρёŉ ĩи Windows Τёяmīńаł Ρřένĭ℮ŵ !!! !!! !!!
TerminalApp ShellExtension_OpenInTerminalMenuItem_Preview Òφèñ ìń Windows Ţ℮ŗмїņāŀ Ρřзνі℮ώ !!! !!! !!!
TerminalApp ShellExtension_OpenInTerminalMenuItem_Preview Оρзñ їñ Windows Τéгмĭлдļ Ρŗěνΐėш !!! !!! !!!
TerminalApp ShellExtension_OpenInTerminalMenuItem_Preview Открыть в ознакомительной версии Терминала Windows
TerminalApp ShellExtension_OpenInTerminalMenuItem_Preview 在 Windows 终端 预览版中打开
TerminalApp ShellExtension_OpenInTerminalMenuItem_Preview 使用 Windows 終端機預覽開啟
```

Spanish (Spain) is the only language that contains this mistake. 🤣 
Sorry @rubenalamina!

**Comment by rubenalamina** [comment]
Good to know it was not me and it's something as simple as that. Looking forward to not having the context menu that wide to accommodate for the longer text. 🤣

**Comment by DHowett** [comment]
I believe this was fixed/changed with #12090
